zigzag/link_generation_facade.py

```diff
@@ -80,6 +80,34 @@ def github_testlog_failure_link(self, test_log):
         except AttributeError:
             return 'Unknown'  # If we ask for the failure link and can't determine it we will supply 'Unknown'
 
+    def github_diff_link(self, pass_fork, pass_base):
+        """Generates a GitHub compare link based on the attributes of this facade
+        This method would be used when we have a last known pass of a given test
+        We are making an assumption that the attributes of this facade are children of
+        pass_fork and pass_base
+
+        Args:
+            pass_fork (str): the fork you wish to compare against
+            pass_base (str): the branch or SHA you want to compare against
```

Comment:
The "pass" prefix is confusing for these arguments. You never test for whether the fork is passing or not. I suggest just using "fork" and "base" for argument names.